src/ts/background/store/wallet/reducer.ts

```diff
@@ -4,15 +4,18 @@ import { ACTION_TYPES, IAccount } from './actions'
 export interface IWallet {
   accounts?: IAccount[],
   created: boolean,
+  isCreatingAccount?: boolean,
   locked: boolean,
   newPassword?: string,
   newPhrase?: string,
   newAccountName?: string,
+  expiryTimeLeft: number
 }
 
 const initialState: IWallet = {
   locked: true,
-  created: false
+  created: false,
+  expiryTimeLeft: 0
```

Comment:
I would call it more specific e.g`tempPwdExpiryTimeLeft`

Reply:
It can either be temp password or the mnemonic. If you'd like it to be more specific, then we'd need to create 2 separate variables to keep track of which seems unnecessary to me. Does that make sense?

Reply:
right now, if i look at it, i would guess it is exprit time for wallet.

Reply:
True. But all the account setup related variables are within `Wallet` state. Hence I've added `expiryTimeLeft` to its state.

I can add a comment beside `expiryTimeLeft` mentioning that it's while setting up an account. If you'd like me to refactor the variable name across the codebase to something else like `expiryTimeLeftForAccountSetup`, I can do that as well.

Reply:
Is it the same expiry time left for both password and mnemonic?

Reply:
Yep. Because the password timer expiry will only happen once at the time of setting up the wallet for first time during which it's valid for mnemonic as well.

Later, when always setting up a new account, the `expiryTimerLeft` will only be applicable to mnemonic since the wallet has been setup already.

Reply:
I see. In this case, we probably can call it `expiryTimeLeftForAccountSetup` as you suggested.

Reply:
Should I do the above refactor throughout the codebase including the methods like `getExpiryTimeLeft` as well as in the `keyring-vault.ts` like `_expiryTimeLeftTimerId`?

Honestly, `expiryTimeLeftForAccountSetup` is a pretty long variable name and I'm more biased towards having a comment that explains its usage rather than refactoring across. I'll leave it upto you though.

Reply:
@anjj123 @fyang1024 @mytechtip any thought on this?

Reply:
Any update on this? Thanks!

Reply:
How about calling it `timeoutForAccountSetup` or `accountSetupTimeout`?

Reply:
@mytechtip - Sure, I'll update it to `accountSetupTimeout `. Although, I'd still keep the `expiryTimeLeft` in the `keyring-vault.ts` and add a little comment beside mentioning that this timer would be for password/mnemonic expiry. Does that make sense?